gui/wxpython/gmodeler/model.py

```diff
@@ -950,9 +950,8 @@ def GetRelations(self, fdir=None):
             if fdir == "from":
                 if rel.GetFrom() == self:
                     result.append(rel)
-            else:
-                if rel.GetTo() == self:
-                    result.append(rel)
+            elif rel.GetTo() == self:
```

Comment:
Lines 950 and 951 could be merged to (including line 952):
```
            if fdir == "from" and rel.GetFrom() == self:
                result.append(rel)
```

Reply:
Probably to apply

Reply:
This suggestion ended up causing a regression.